neo/Network/P2P/Peer.cs

```diff
@@ -24,7 +24,7 @@ public class Connect { public IPEndPoint EndPoint; public bool IsTrusted = false
         private class Timer { }
         private class WsConnected { public WebSocket Socket; public IPEndPoint Remote; public IPEndPoint Local; }
 
-        public const int DefaultMinDesiredConnections = 10;
+        public const int DefaultMinDesiredConnections = 20;
```

Comment:
Does this field also affect RPC?

Reply:
I don't think so

Reply:
> Does this field also affect RPC?

If MaxConcurrentConnections is missing or equals 0 in config.json, then `DefaultMinDesiredConnections*4` will be the default value. Otherwise, 40 peers limit is too less for p2p connections now in mainnet especially for seed nodes, which has already made other nodes too difficult to join NEO network. I've already rasied it to 80 connections which makes mainnet stable for joining now.